pkg/api/mlflow/service/artifact/storage/s3.go

```diff
@@ -121,3 +125,27 @@ func (s S3) List(artifactURI, path string) ([]ArtifactObject, error) {
 
 	return artifactList, nil
 }
+
+// Get returns file content at the storage location.
+func (s S3) Get(artifactURI, itemPath string) (io.ReadCloser, error) {
+	bucketName, prefix, err := ExtractS3BucketAndPrefix(artifactURI)
+	if err != nil {
+		return nil, eris.Wrap(err, "error extracting bucket and prefix from provided uri")
+	}
+
+	input := &s3.GetObjectInput{
+		Bucket: aws.String(bucketName),
+		Key:    aws.String(filepath.Join(prefix, itemPath)),
+	}
+
+	resp, err := s.client.GetObject(context.TODO(), input)
+	if err != nil {
+		var s3NoSuchKey *types.NoSuchKey
```

Comment:
you have a pointer here and the take address?

Reply:
yes that is the example in documentation from aws sdk